packages/unlock-js/src/PublicLock/v10/index.js

```diff
@@ -1,5 +1,6 @@
 import abis from '../../abis'
 import purchaseKeys from './purchaseKeys'
+import purchaseKey from './purchaseKey'
```

Comment:
I added both batch and single purchase of keys. The idea was to make it simpler on frontend. Mot sure if both are needed tho ?

Reply:
That's a good question.
From my perspective, I want to make it as easy on the front-end to buy 1 or multiple keys for _all_ lock versions, so ideally we will move everything to just use `purchaseKeys` everywhere but of course it will just work for older version if there is a single key and would call `purchaseKey` internally.